packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -496,14 +560,21 @@ export default class StatsAnalyzer extends EventsScope {
             LoggerProxy.logger.info(`StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames received`);
           }
 
-          if (this.statsResults.resolutions[mediaType].recv.framesDecoded === this.lastStatsResults.resolutions[mediaType].recv.framesDecoded || this.statsResults.resolutions[mediaType].send.framesDecoded === 0) {
+          if (this.statsResults[mediaType].recv.framesDecoded === this.lastStatsResults[mediaType].recv.framesDecoded || this.statsResults.resolutions[mediaType].send.framesDecoded === 0) {
```

Comment:
I've changed this, because in Firefox `this.statsResults.resolutions[mediaType].recv.framesDecoded` was undefined but `this.statsResults[mediaType].recv.framesDecoded` is available in all the browsers.